src/Microsoft.AspNetCore.Mvc.Core/Internal/ITypeActivatorCache.cs

```diff
@@ -15,8 +15,8 @@ public interface ITypeActivatorCache
         /// Creates an instance of <typeparamref name="TInstance"/>.
         /// </summary>
         /// <param name="serviceProvider">The <see cref="IServiceProvider"/> used to resolve dependencies for
-        /// <paramref name="implementationType"/>.</param>
-        /// <param name="implementationType">The <see cref="Type"/> of the <typeparamref name="TInstance"/> to create.</param>
+        /// <paramref name="optionType"/>.</param>
```

Comment:
I don't understand your concern. This is a `paramref`. The `param` tag is below this.